### PR TITLE
Enhance style guide

### PR DESCRIPTION
**Pull request by krlmlr**
Fixes link.

### CODE REVIEW

**Comment by krlmlr** [comment]
@hadley: I've looked at violations of the style guide in dplyr's sources, this is the summary. No other PRs touch this file, so this is safe to merge (after we agree on the contents, of course).

**Comment by hadley** [comment]
I'm planning to pull out to a separate book down site so adv-r, r-pkgs, and r4ds can all share a common resource and there's more space for details. Would you mind opening an issue on r4ds pointing to this PR so I don't forget about it?

**Comment by krlmlr** [comment]
Done. We could still discuss/merge this PR so that we can simply point contributors to this document.

File: style.rmd [hunk 1]
```diff
@@ -164,6 +161,26 @@ It's ok to leave very short statements on the same line:
 if (y < 0 && debug) message("Y is negative")
 ```
 
+### Pipes
+
+If you use the `%>%` operator from the tidyverse, put each verb on its own line.  This makes it simpler to rearrange them later, and makes it harder to overlook a step.
```
Comment (hadley) [inline style.rmd]: Do we want to say to only use the pipe if you have more than one step?

File: style.rmd [hunk 2]
```diff
@@ -175,11 +192,31 @@ When indenting your code, use two spaces. Never use tabs or mix tabs and spaces.
 The only exception is if a function definition runs over multiple lines. In that case, indent the second line to where the definition starts:
 
 ```{r, eval = FALSE}
+# Good
+do_something_very_complicated("that", requires, many, arguments,
+  "some of which may be long")
+
+do_something_very_complicated(
```
Comment (hadley) [inline style.rmd]: I think this maybe needs it's own section?

File: style.rmd [hunk 3]
```diff
@@ -175,11 +192,31 @@ When indenting your code, use two spaces. Never use tabs or mix tabs and spaces.
 The only exception is if a function definition runs over multiple lines. In that case, indent the second line to where the definition starts:
 
 ```{r, eval = FALSE}
+# Good
+do_something_very_complicated("that", requires, many, arguments,
```
Comment (hadley) [inline style.rmd]: Should be 

```R
do_something_very_complicated(
  "that", requires, many, arguments,
  "some of which may be long")
)
```

But typically you'd put one named argument on each line:

```R
do_something_very_complicated(
  a = 1,
  b = 2,
  thisisalongarugment = 3
)
```

Comment (hadley) [inline style.rmd]: But those are two different styles of function - the first is `...` style like `paste()`, and the second is one where arguments are named.

**Comment by krlmlr** [comment]
@hadley: Would you consider merging this for a more complete style guide?

**Comment by hadley** [comment]
Sorry, slipped my mind.

**Pull request merged**